src/apps/content-editor/src/app/views/ItemList/TableCells/OneToManyCell.tsx

```diff
@@ -42,13 +42,22 @@ export const OneToManyCell = ({ items }: OneToManyCellProps) => {
   const [lastValidIndex, setLastValidIndex] = useState(contentCount - 1);
   const parentWidth = chipContainerRef.current?.parentElement?.clientWidth;
   const hiddenItems = items?.length - lastValidIndex - 1;
+  const dispatch = useDispatch();
 
   useEffect(() => {
     setLastValidIndex(
       getNumOfItemsToRender(parentWidth, chipContainerRef.current?.children)
     );
   }, [parentWidth]);
 
+  useEffect(() => {
```

Comment:
Won't this exasperate performance? As this does not check the redux stores and instead makes an API request for every item listed every time the view loads.